VisualPinball.Unity/VisualPinball.Unity.Editor/VPT/PhysicsMaterialInspector.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Visual Pinball Engine
+// Copyright (C) 2022 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// ReSharper disable AssignmentInConditionalExpression
+
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using VisualPinball.Engine.VPT;
+
+namespace VisualPinball.Unity.Editor
+{
+	[CustomEditor(typeof(PhysicsMaterialComponent)), CanEditMultipleObjects]
+	public class PhysicsMaterialInspector : UnityEditor.Editor
+	{
+		SerializedProperty Elasticity;
+		SerializedProperty ElasticityFalloff;
+		SerializedProperty ElasticityOverVelocity;
+		SerializedProperty Friction;
+		SerializedProperty FrictionOverVelocity;
+		//SerializedProperty FrictionOverAngularMomentum;
+
+		SerializedProperty ScatterAngle;
+
+
+		private bool _foldoutDebug = true;
+		private bool _foldoutColliders;
+		private string[] _currentColliders;
+		private Vector2 _scrollPos;
+
+		//protected override MonoBehaviour UndoTarget => null;
+
+
+		private void OnEnable()
+		{
+			Elasticity = serializedObject.FindProperty("Elasticity");
+			ElasticityFalloff = serializedObject.FindProperty("ElasticityFalloff");
+			ElasticityOverVelocity = serializedObject.FindProperty("ElasticityOverVelocity");
+			Friction = serializedObject.FindProperty("Friction");
+			ScatterAngle = serializedObject.FindProperty("ScatterAngle");
+			FrictionOverVelocity = serializedObject.FindProperty("FrictionOverVelocity");
```

Comment:
I would prefer using `nameof(PhysicsMaterialComponent.Elasticity)`, which is more refactor-friendly.

Reply:
Oh, yes. did not know that this works. Great.